src/components/Pill/Pill.scss

```diff
@@ -0,0 +1,15 @@
+@import '~@quartz/styles/scss/color-scheme';
+@import '~@quartz/styles/scss/fonts';
+@import '~@quartz/styles/scss/media-queries';
+
+.container {
+	@include font-maison-extended-700-1;
+
+	background-color: $color-border-decorative;
+	border-radius: 0.75em;
```

Comment:
should this use the `$size-border-radius` token?

Reply:
that doesn't match design, but i'll query the daniels-that-be